docs/content/releases.md

```diff
@@ -10,6 +10,7 @@ docs: "DOCS-616"
 29 Mar 2023
 
 OVERVIEW:
+* **IMPORTANT** Beginning with release 3.1.0 the NET_BIND_SERVICE capability is no longer used, and instead relies on net.ipv4.ip_unprivileged_port_start sysctl to allow port binding. Kubernetes 1.22 or later is required for this sysctl to be [classified as safe](https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#safe-and-unsafe-sysctls). **Ensure** that you are using the latest updated `deployment` and `daemonset` examle yaml files available in the repo.
 * *The minimum supported version of Kubernetes is now 1.22*. The NGINX Ingress Controller now uses `sysctls` to [bind to lower level ports without privilege escalation](https://github.com/nginxinc/kubernetes-ingress/pull/3573/). This removes the need to use `NET_BIND_SERVICE` to bind to these ports. Thanks to [Valters Jansons](https://github.com/sigv) for making this feature possible!
```

Comment:
"bind to lower level ports without privilege escalation" should be
"bind to privileged ports without privilege escalation"
yes, it sounds redundant but the accurate terms are "privileged ports", and "privilege escalation"